docs/main_page.md

```diff
@@ -1,63 +1,72 @@
 # Network Layer for RF24 Radios
+
 This class implements an [OSI Network Layer](http://en.wikipedia.org/wiki/Network_layer) using nRF24L01(+) radios driven
 by the newly optimized [RF24 library fork](http://nRF24.github.com/RF24/).
 
-@see http://nRF24.github.io/RF24/index.html for general RF24 configuration and setup
-
-@see Also, [Linux Installation](http://nRF24.github.io/RF24/md_docs_linux_install.html) and [General Linux/RPi configuration and setup](http://nRF24.github.io/RF24/md_docs_rpi_general.html) documentation
+@see
+[RF24 Library docs](http://nRF24.github.io/RF24/) for general RF24 configuration and setup.
+    - [Linux Installation](http://nRF24.github.io/RF24/md_docs_linux_install.html) and [General Linux/RPi configuration and setup](http://nRF24.github.io/RF24/md_docs_rpi_general.html) documentation
```

Comment:
These links will lead users to the build system they prefer. _RF24/md_docs_linux_install.html_ has an advisory at the top of the page now. Its not explicit in this lib's docs, but then again, it never really was.